src/notation/view/editpercussionshortcutmodel.cpp

```diff
@@ -56,6 +56,13 @@ void EditPercussionShortcutModel::inputKey(Qt::Key key)
         return;
     }
 
+    switch (key) {
+    case Qt::Key_Enter:
+    case Qt::Key_Return:
+        return;
+    default: break;
+    }
+
```

Comment:
you can move it and `shortcuts::needIgnoreKey` to separate method needIgnoreKey, will be more clean 
also use `set` of keys as in `shortcuts::needIgnoreKey` :)